test/azure/templates/azuredeploy.json

```diff
@@ -14,6 +24,34 @@
           ]
         }
       }
+    },
+    {
```

Comment:
keep for now & we can touch later

I think what actually want to do is deploy the same template 2 times - once per RG (in parallel if we want)

**or**

Change the template to deploy at subscription scope, and that template stamps out both the 2 RG's and their contents in a consistent manner